package.json

```diff
@@ -73,7 +74,7 @@
     "eslint": "^8.32.0",
     "eslint-config-prettier": "^8.6.0",
     "husky": "^8.0.3",
-    "lerna": "^7.1.1",
+    "lerna": "^7.4.2",
     "lint-staged": "^13.1.0",
```

Comment:
I had to force install lerna again to fix the issue i had with publishing packages